AdobeStockImageAdminUi/Test/Mftf/Test/AdminAdobeStockPreviewNotSavedLicensedImageSaveTest.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminAdobeStockPreviewNotSavedLicensedImageSaveTest">
+        <annotations>
+            <features value="AdobeStockImagePanel"/>
+            <stories value="User saves licensed image into Magento Media Gallery"/>
+            <title value="Adobe Stock Licensed Image Save"/>
+            <description value="User saves previously licensed image into Magento Media Gallery"/>
+            <severity value="CRITICAL"/>
+            <group value="adobe_stock_integration_license"/>
+            <group value="adobe_stock_integration"/>
+        </annotations>
+        <before>
+            <actionGroup ref="LoginAsAdmin" stepKey="loginAsAdmin"/>
+            <actionGroup ref="AdminOpenMediaGalleryForPageNoEditorActionGroup" stepKey="openMediaGalleryForPage"/>
+            <actionGroup ref="AdminAdobeStockOpenPanelFromMediaGalleryActionGroup" stepKey="openAdobeStockPanel"/>
+            <actionGroup ref="AdminAdobeStockClickSignInActionGroup" stepKey="clickOnSignIn"/>
+            <actionGroup ref="AdminAdobeStockImsPopupSignInFillUserDataActionGroup" stepKey="fillUserCredentials"/>
+            <actionGroup ref="AdminAdobeStockImsPopupClickSignInActionGroup" stepKey="clickSignInImsPopup"/>
+            <actionGroup ref="AdminAdobeStockAssertUserLoggedActionGroup" stepKey="assertUserLoggedIn"/>
+        </before>
+        <after>
+            <actionGroup ref="AdminMediaGalleryDeleteImage" stepKey="deleteImageActionGroup">
+                <argument name="generatedImageName" value="//NEED TO PASS VALUE HERE"/>
```

Comment:
Hi @sivaschenko, I need to pass the generated image name here to delete. And this test is when the preview image is not previously saved. So the generated image name is in another action group (AdminAssertAdobeStockImageSavedActionGroup). I don't think I can accommodate the image delete in that ActionGroup since it is used in multiple places. So do I create a new ActionGroup which will have the same functionality as the one mentioned above, but deletes the image after asserting that it is present in the Media Gallery

Reply:
@filmaj could I get some guidance here, please? Thank you.